WoD/BloodmaulSlagMines/Crushto.lua

```diff
@@ -52,6 +54,10 @@ function mod:WildSlam(args)
 	self:MessageOld(args.spellId, "yellow", "long")
 end
 
+function mod:EarthCrush(args)
+	self:MessageOld(args.spellId, "yellow", "long")
```

Comment:
Probably `alarm` is a good default sound here - `long` is generally used for stage changes or rarely for abilities with long cast times / channels. `alarm` is a bit more general purpose but often used for frontals/swirlies/etc.

While this file does still use MessageOld (and it still works), it's better practice nowadays to separate the message and the sound to different lines for clarity. So this could be written as:

```
self:Message(args.spellId, "yellow")
self:PlaySound(args.spellId, "alarm")
```

Reply:
I really don't know how to write lua, I just copied the existing code and adapted it. Gonna see if I can make sense of your suggestions though.

Reply:
What I truly don't get is how the "long" sound option (even if it may not be the best suited) works fine for the "WildSlam" funtion but not "EarthCrush"?

Reply:
Because Wild Slam has an entry in [Sounds.lua](https://github.com/BigWigsMods/LittleWigs/blob/master/WoD/BloodmaulSlagMines/Options/Sounds.lua#L11) and Earth Crush does not.

Reply:
Ah yes, I understand. Thanks. Can I add additional  LUA to one pull request then, or do I have to make a separate request for the altered Sounds.lua? Sorry for bombarding you with questions, I really don't know what I'm doing but I'm trying to understand and want to be able to contribute in the future. :)

Reply:
Yes, you can just add the changes to the branch you used for this pull request.

Reply:
Well, I've done what I can on my end. Don't know why the LUA error appears, on my end all brackets seem closed. Also, I didn't realised every change I made on my fork got pushed onto this Pull Request, so I'm guessing someone is really annoyed with my fiddling around since I thought I was only making local changes.

Reply:
It's not every change on the fork, just the changes committed to the branch the PR was opened from - so no worries.

The error was from a missing comma, but I fixed that before merging it. Thanks for making the changes.